R/snapshots.R

```diff
@@ -124,12 +132,21 @@ convertSnapshot <- function(..., format, output = ".") {
   }
 
   cli::cli_process_start(
-    msg = "Converting {nfiles} files{?s} to {format} format",
+    msg = "Converting {nfiles} file{?s} to {format} format",
     msg_done = "Conversion completed",
     msg_failed = "An error occured while converting files"
   )
 
-  rSharp::callStatic("PKSim.R.Api", "RunSnapshot", SnapshotRunOptions)
+  tryCatch(
+    {
+      invisible(rSharp::callStatic("PKSim.R.Api", "RunSnapshot", SnapshotRunOptions))
+    },
+    error = function(e) {
+      message <- stringr::str_extract(as.character(e), "(?<=Message: )[^\\n]*")
```

Comment:
Some error messages won't have a message line of format "Message: *". In this case `str_extract()` returns in NA resulting in 
```
Error in `convertSnapshot()`:
! NA
✖ An error occured while converting files
```

Reply:
Good catch, I added an if case to handle this.